hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/HoodieROTablePathFilter.java

```diff
@@ -231,6 +240,12 @@ public boolean accept(Path path) {
     }
   }
 
+  @VisibleForTesting
```

Comment:
As CheckStyle doesn't encourage to import `com.google.common.annotations.VisibleForTesting` annotation. I simply follow `metaClientCache` to make `nonHoodiePathCache` access specifier is `friendly` for test purpose.